addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -4960,7 +4960,8 @@ registry.layout_column = SnippetOptionWidget.extend({
 
         if (elementType === 'image') {
             // Set the columns properties.
-            newColumnEl.classList.add('col-lg-6', 'g-col-lg-6', 'g-height-6');
+            newColumnEl.classList.add('col-lg-6', 'g-col-lg-6', 'g-height-6', 'o_grid_item_image');
+            newColumnEl.contentEditable = false;
```

Comment:
That is because it was the image that was selected. Even in normal mode, you have to click next to the image to make the blinking cursor appear and so to not replace the image. Here, this line is just to prevent to write text inside the column (so to not have the cursor). However, it unfortunately doesn't prevent the image to be removed/replaced if this is what was selected. 
I tried to find a solution to make the image "unremovable" but I didn't find one :/ (for example by preventing the user to select the image with `user-select: none`. With that, the blue highlight doesn't show (only on Chrome though...) but the image is still removable :/)
Do you have any suggestion ?

Note 1: the image is highlighted in blue when it is selected so we have an indication of that.
Note 2: the problem in Firefox is that the text appears inside the row but outside of any column (in Chrome the text goes in the previous column). I didn't have any crash but this is still problematic because it is incorrectly placed in the grid.

Knowing all that, what is the best ? Not editing but having weird effects if we don't see the selection highlight, or allowing the edition but having the ugly overlay overflow effect there was before ? (If you have no suggestion of course)

Reply:
> Do you have any suggestion ?

Not really, typing text at wrong locations has always been a problem that the editor team will likely solve the right way at some point. Meanwhile, maybe the class oe_unremovable (something like that) could prevent the image to be removed... but I think it would also prevent removing the image via the trash icon etc so... probably not great either :grimacing: I'll have to test better to understand/remember the real issue here but I am tempted to say "let's let people break stuff". As long as there is no traceback, letting people write at wrong places is not a big deal to me as they can always go back (normally) or type at the right places. But maybe I don't remember fully the issue that is being solved, we can talk about it next week.